test/models/trip_test.rb

```diff
@@ -1,24 +1,49 @@
 require "test_helper"
+require "pry"
 
 describe Trip do
+  before do
+    @new_driver = Driver.create(name: "TEST123", vin: "WBWSS52P9NEYLVDE9", available: true)
+    @new_passenger = Passenger.create(name: "Judy", phone_num: "360-555-0987")
+  end
+
+    let (:new_trip) {
+      Trip.create(driver_id: @new_driver.id, passenger_id: @new_passenger.id, date: "2016-04-05", rating: 3, cost: 1293)
+    }
+
   it "can be instantiated" do
-    # Your code here
+    expect(new_trip.valid?).must_equal true
   end
 
   it "will have the required fields" do
-    # Your code here
+    new_trip.save
+    trip = Trip.first
+    [:driver_id, :passenger_id, :date, :rating, :cost].each do |field|
+      # Assert
+      expect(trip).must_respond_to field
+    end
   end
 
   describe "relationships" do
```

Comment:
I would also include a test that you can't create a trip without a passenger and driver.